RadixWallet/Clients/BackupsClient/BackupsClient+Interface.swift

```diff
@@ -31,11 +34,10 @@ public struct BackupsClient: Sendable {
 extension BackupsClient {
 	public typealias SnapshotOfProfileForExport = @Sendable () async throws -> Profile
 	public typealias LoadProfileBackups = @Sendable () async -> Profile.HeaderList?
-
+	public typealias LookupProfileSnapshotByHeader = @Sendable (Profile.Header) async throws -> Profile?
 	public typealias ImportProfileSnapshot = @Sendable (Profile, Set<FactorSourceIDFromHash>) async throws -> Void
+	public typealias DidExportProfileSnapshot = @Sendable (Profile) throws -> Void
```

Comment:
```suggestion
	public typealias DidExportProfile = @Sendable (Profile) throws -> Void
```

to follow the pattern on the changes done [here](https://github.com/radixdlt/babylon-wallet-ios/pull/1114/files#diff-36d3a26d72311238ab6bd57ce5b40bf982f396903cafaf1c9faa97360312bd10R15)

Reply:
yes

Reply:
But there are lots of other places where we still call it snapshot, including the other APIs on this client, so let's fix that all later and avoid merge issues.

Reply:
Yes but this is a new method, why add it with an outdated name (`Snapshot`) when we can have it correctly?

Reply:
Does it matter? The purpose of this PR is too let you build your stuff on top of it.

Reply:
The import function still uses "Snapshot"

Reply:
yeah, just wanted to avoid merging code that we already want to change.. but we can fix later